javascript-stringify.js

```diff
@@ -229,6 +229,11 @@
       return PRIMITIVE_TYPES[typeof value](value, indent, next);
     }
 
+    // Handle Error objects.
+    if (value instanceof Error) {
```

Comment:
yes, but `react-json-tree` already expands Error objects by itself. We're using it for [showing diffs as a string in `redux-devtools-inspector`](https://github.com/alexkuz/redux-devtools-inspector/blob/780116c94ded5d868e9623d850221824b0cbdff0/src/tabs/JSONDiff.jsx#L7-L14). Skipping Error objects leads to issue like https://github.com/alexkuz/redux-devtools-inspector/issues/42.

Also I'm using it for showing data in [`redux-devtools-test-generator`](https://github.com/zalmoxisus/redux-devtools-test-generator), and intend to add in [`redux-devtools-chart-monitor`](https://github.com/romseguy/redux-devtools-chart-monitor) for showing data in the tooltip. It offers a better data visualization (`{a: 'smth'}` is better than `{"a": "smth"}` and allows to see unserializable by `JSON.stringify` data. I'm not using it for stringifying/parsing data sent to [Chrome/Firefox extension](https://github.com/zalmoxisus/redux-devtools-extension), because want to avoid `eval`, which would require special permissions in the manifest.